deployments/.gitignore

```diff
@@ -0,0 +1,3 @@
+abis/
+dist/
+*.tgz
```

Comment:
We can remove `abis` and `*.tgz` here because everything is generated inside `dist`, right?